.github/workflows/rollback.yml

```
@@ -0,0 +1,250 @@
+        with:
+          target: wasm32-unknown-unknown
+
+      - name: Install Soroban CLI
+        run: cargo install --locked soroban-cli --features opt
+
+      - name: Rollback contracts
+        env:
+          SOROBAN_ACCOUNT_SECRET: ${{ secrets.SOROBAN_ACCOUNT_SECRET }}
+          NETWORK: ${{ inputs.environment == 'production' && 'public' || 'testnet' }}
+          RPC_URL: ${{ inputs.environment == 'production' && 'https://soroban-mainnet.stellar.org' || 'https://soroban-testnet.stellar.org' }}
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:5:31: Double quote to prevent globbing and word splitting

(shellcheck)